docs/running-on-mesos.md

```diff
@@ -211,6 +211,14 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
 </tr>
 <tr>
+  <td><code>spark.mesos.executor.cores</code></td>
+  <td>1.0</td>
+  <td>
+    Set the amount of cores to request for running a mesos executor. This setting is only used for Mesos fine-grained mode.
+    By default, executor will use the amount of cores even though no task is running on an executor.
```

Comment:
I would change this to "The setting, which can be a floating point number, controls the number of cores allocated to the executor not for use by tasks."